Tests/UnitTests/Core/Seeding/SeedfinderTest.cpp

```diff
@@ -134,10 +134,14 @@ int main(int argc, char** argv) {
   config.beamPos = {-.5_mm, -.5_mm};
   config.impactMax = 10._mm;
 
+  config.numPhiNeighbors = 1;
+
   auto bottomBinFinder = std::make_shared<Acts::BinFinder<SpacePoint>>(
-      Acts::BinFinder<SpacePoint>());
+      Acts::BinFinder<SpacePoint>(std::move(config.zBinNeighborsBottom),
+                                  std::move(config.numPhiNeighbors)));
   auto topBinFinder = std::make_shared<Acts::BinFinder<SpacePoint>>(
-      Acts::BinFinder<SpacePoint>());
+      Acts::BinFinder<SpacePoint>(std::move(config.zBinNeighborsTop),
+                                  std::move(config.numPhiNeighbors)));
```

Comment:
please don't move individual parts out of a config that you continue to use later. By std::move-ing them, they members of the config struct become invalid.

as mentioned above, you don't need to have the neighbor bins in the SeedfinderConfig at all, you can just create them separately.